### PR TITLE
Update html to reflect changes to contrib.

### PR DESCRIPTION
**Pull request by cshewmake2**

### CODE REVIEW

File: _sources/contributing.rst.txt [hunk 1]
```diff
@@ -421,18 +475,52 @@ When writing docstrings, follow the `NumPy template <https://numpydoc.readthedoc
...
+ 
    2. Use ``[`` for defining shapes: ``array-like, shape=[n_samples,]``
+ 
    3. For strings with multiple options, use brackets:
       ``input: str, {'log', 'squared', 'multinomial'}``
+ 
    4. 1D or 2D data can be a subset of
       ``{array-like, ndarray, sparse matrix, dataframe}``. Note that ``array-like``
       can also be a ``list``, while ``ndarray`` is explicitly only a ``numpy.ndarray``.
+
```
Comment (ninamiolane) [inline _sources/contributing.rst.txt:523]: Maybe also add the comments from our discussion, to have it 100% clear?

- Description and parameters' description starts with uppercase, ends with a .

- optional is on the same line as the parameters' type.

- There are spaces around :

- The description of a method starts with an imperative verb, OK to use Compute if needed (but not, Calculate, Evaluate, etc, let's agree on Compute).

**Review by ninamiolane — APPROVED** [review]
Thanks, just a small comment.

File: _sources/contributing.rst.txt (hunk 1 above)
Comment (cshewmake2) [reply _sources/contributing.rst.txt:523]: - Description beginning with uppercase and ending with period is in the current version. I'll add it to the Parameters section's description as well.

- Thanks, I'll add this (optional info)

- Thanks, I'll add this (`` : ``)

- I don't think the Description section has to start with an imperative verb, only the Summary section. Also, this point (imperative verb) is written in the Summary section,  but I'll add the compute vs calculate point next. :)

**Review by ninamiolane — APPROVED** [review]
Thank you!!

**Pull request merged**